src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -16,16 +32,180 @@ interface ComponentProps {
     title: string;
     startDate?: string;
     endDate?: string;
+    activeProject: number;
+    analysisId: number;
+    setAnalysisDeleteId: () => number;
```

Comment:
is this the correct type for function? Does the function return number?  I think it should be 
```suggestion
    setAnalysisDeleteId: (value: number) => void;
```